### PR TITLE
Improved geometry IsValid check

### PR DESCRIPTION
**Pull request by joto**
Always use trigger and don't check point geometries (in flex output).

### CODE REVIEW

File: tests/test-output-flex-schema.cpp [hunk 1]
```diff
@@ -31,12 +31,12 @@ TEST_CASE("config with schema should work")
...
 
-    REQUIRE(1 ==
+    REQUIRE(0 ==
             conn.get_count("pg_proc",
                            "proname = 'osm2pgsql_test_point_osm2pgsql_valid'"));
 
-    REQUIRE(1 == conn.get_count("pg_trigger"));
-    REQUIRE(1 ==
+    REQUIRE(0 == conn.get_count("pg_trigger"));
+    REQUIRE(0 ==
```
Comment (lonvia) [inline tests/test-output-flex-schema.cpp:39]: So you don't test at all anymore that the trigger ends up in the right schema?

**Review by lonvia — COMMENTED** [review]
Should the trigger be dropped when --drop is given? That would correspond to the previous behaviour.

File: tests/test-output-flex-schema.cpp (hunk 1 above)
Comment (joto) [reply tests/test-output-flex-schema.cpp]: Ups. Using the linestring table now fixes this.

**Comment by joto** [comment]
Function/trigger is now dropped correctly.

**Pull request merged**